trl/mergekit_utils.py

```diff
@@ -0,0 +1,347 @@
+import os
+import torch
+
+from mergekit.config import MergeConfiguration
+from mergekit.merge import MergeOptions, run_merge
+
+
+from huggingface_hub import HfApi
+
+
+def get_last_checkpoint_path(output_dir):
```

Comment:
Can we reuse the `get_last_checkpoint()` [method](https://github.com/huggingface/transformers/blob/a3d69a8994d673899608a7c17fbf4f953f50474e/src/transformers/trainer_utils.py#L212) from `transformers` or is checking for the last modification time necessary here?

Reply:
just replaced it